sandbox/components/PageHeader.vue

```diff
@@ -36,7 +40,21 @@ const query = ref('')
 
 watch(query, (searchQuery: string) => {
   emit('search', searchQuery)
+
+  if (searchQuery) {
+    if (searchQuery !== route.query.search) {
+      router.push({ name: 'home', query: { search: searchQuery } })
+    }
+  } else {
+    router.push({ name: 'home' })
```

Comment:
Is‘t it going to send me to home page if both query string and search query is same?

Reply:
Yes and now. It does save the search query in input field but removes the query from route (screenshot)

![Screen Shot 2023-09-20 at 3 56 43 PM](https://github.com/Kong/icons/assets/36751160/da744c1b-1400-4c05-992d-0f2ec994b450)

Reply:
Let's merge this already 😅